src/pcrlock/pcrlock.c

```diff
@@ -4724,6 +4724,16 @@ static int verb_make_policy(int argc, char *argv[], void *userdata) {
         return make_policy(arg_force, arg_recovery_pin);
 }
 
+static int verb_can_make_policy(int argc, char *argv[], void *userdata) {
+        _cleanup_(tpm2_context_unrefp) Tpm2Context *tc = NULL;
+        int r = tpm2_context_new(NULL, &tc);
+        if (r < 0)
+                return log_error_errno(r, "Failed to allocate TPM2 context: %m");
```

Comment:
"allocate" sounds as if this was an OOM issue. The function actually connects to the TPM, hence maybe say "Failed to open TPM2 context:"?

Reply:
c&p from another place in the same file :) will refactor